brainstormforce/ultimate-addons-for-gutenberg
# Remove jQuery dependency for Post masonry and carousel.

### Description

- Masonry block.
- Carousel block.

### Types of changes




- Improvement

### How has this been tested?

- Drag and drop masonry 
- Enable pagination and check infinite scroll and load more button pagination settings.
- Drag and drop carousel
- Enable equal height check editor as well as frontend. Equal hight working properly or not.

### Checklist:
